charts/meteor-pipelines/templates/validate-jupyterhub-image.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: validate-jupyterhub-image
```

Comment:
also.. let's (for now) rename this to `byon-validate-jupyterhub-image` (as well as the file name) - so it's more easy to recognize it among the other pipelines/tasks

Reply:
Resolved in latest commit.